gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=b13f5d97f08000996bf12d9dd70af3f2c6b694c2c663ab1b545e9695562ad1ee
```

Comment:
`distributionSha256Sum` must be updated. We use it to ensure the integrity of Gradle distributions.

Reply:
Wrapper upgrade removed this.

Reply:
I believe it's this value:
https://services.gradle.org/distributions/gradle-7.3.3-all.zip.sha256